ci_framework/roles/oc_cli/tasks/main.yml

```diff
@@ -0,0 +1,66 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+-
+  name: Ensuring the supported version of oc is present
+  block:
+    -
+      name: Gather OpenShift Client version
```

Comment:
note that you probably want to pass this:
```
environment:
  PATH: "{{ cifmw_path }}"
```

That fact is either customized by the user, or built from within ci_setup role, and would allow to use the `oc` from crc for instance.